realm/realm-library/src/objectServer/java/io/realm/internal/objectserver/SyncSession.java

```diff
@@ -204,13 +201,6 @@ public synchronized void onError(ObjectServerError error) {
         }
     }
 
-    // Called from Session.cpp and SyncMaanger
-    // This callback will happen on the thread running the Sync Client.
-    void notifySessionError(int errorCode, String errorMessage) {
```

Comment:
I searched `notifySessionError` in `realm/realm-library/src` and I couldn't find it other than here.